charts/athens-proxy/README.md

```diff
@@ -103,6 +103,17 @@ This will deploy a single Athens instance in the `athens` namespace with `disk`
 | service.type | string | `"ClusterIP"` | Type of service; valid values are "ClusterIP", "LoadBalancer", and "NodePort". "ClusterIP" is sufficient in the case when the Proxy will be used from within the cluster. To expose externally, consider a "NodePort" or "LoadBalancer" service or use an "Ingress". |
 | serviceAccount.annotations | object | `{}` |  |
 | serviceAccount.create | bool | `true` | Create a ServiceAccount |
+| singleFlight.etcd.endpoints | string | `""` |  |
```

Comment:
same here, this table is auto-generated from the values.yaml by "helm-docs"

Reply:
nvm, I just realised that helm-docs obviously does change the readme, so you did everything right 👍